### PR TITLE
Make batteries more standardized

### PR DESCRIPTION
**Pull request by AlexMooney**
#### Summary

```SUMMARY: Bugfixes "Corrects some nonstandard battery slots"```

#### Purpose of change

Noticed that I couldn't reload a cellphone with a disposable light battery.

#### Describe the solution

Review all battery magazines across the json files to correct discrepancies.

#### Describe alternatives you've considered

The `foodperson_mask` was the only example I saw of a light and medium battery compatible item.  I consider this to be a mistake, but the choice between medium and light is fairly arbitrary.  Since the mask is a gigantic 16 liters, I decided to go with medium.

#### Testing

Ran json linting.  Reloaded my cellphone with a light disposable battery.

#### Additional context

The battery lists are not always in the same order from item to item.  I think the proper way to fix that is to overhaul the batteries system to have size flag(s) for items and then adding a new Light battery will propagate naturally.  We could spend the effort to put them in a consistent order now, but I don't think it's worth the time.

Will removing some batteries from the `foodperson_mask` break save games?  I didn't test.

### CODE REVIEW

**Comment by anothersimulacrum** [comment]
> I consider this to be a mistake

The foodperson mask is fairly vital to true foodpeople, so I think this is purposeful, because light batteries are generally easier to acquire than medium batteries, making it easier on beginning true foodpeople. You may want to ask the creator, but if you're going to go for one or the other, I'd suggest light over medium.

**Comment by AlexMooney** [comment]
@Fris0uman can you take a look?  I rebalanced the Foodperson© mask to last 8 hours, 20 minutes with a medium battery.  It doesn't take light batteries now.  Would you prefer something else?

File: data/mods/Graphical_Overmap/fonts.json [hunk 1]
```diff
@@ -1,6 +1,4 @@
 {
-  "typeface": "unifont",
-  "map_typeface": "unifont",
```
Comment (Night-Pryanik) [inline data/mods/Graphical_Overmap/fonts.json]: Unrelated changes?

Comment (AlexMooney) [reply data/mods/Graphical_Overmap/fonts.json]: Ah sorry about that.  I'll rebase it out.  Trying to get that graphical overmap mod working... :fearful:

**Comment by Fris0uman** [comment]
The bigger the battery the longer anyone trying it ON will get trapped, but on the other hand 8h is the duration of a long day shift so that's coherent with the lore of the mask. If you really think it's not acceptable to accept both light and medium battery keep the medium one, otherwise leave it as it was.
